utils/rpms/daos.spec

```
@@ -232,6 +232,10 @@ Requires: fuse3-devel >= 3
 Requires: fuse3-devel >= 3.4.2
 %endif
 Requires: pciutils-devel
+%if (0%{?rhel} >= 8)
+Requires: ndctl-devel
+Requires: daxctl-devel
+%endif
```

Why is this only needed on EL8 and not all distros/versions?

```suggestion
Requires: ndctl-devel
Requires: daxctl-devel
```

---

SuSE detects them automatically.
Only RHEL requires these dependencies.
To be honest - I do not know why.

---

> SuSE detects them automatically.

Hrm.  This must be a side-effect at best and should not be relied upon.  I'm having a hard time in fact imagining how they are being installed.  How do you know they are installed automatically?  I tried to find a CI run with Functional testing on Leap15 and couldn't find any.  I didn't search all 121 runs though, just the most recent 20 or so.

You most certainly should be adding `Skip-func-test-leap15: true` and `Skip-func-test-el9: true` commit pragmas to a PR like this that is making changes to Functional testing dependencies.

---

Why?
The problem has been detected in `Build RPM on Leap 15.5` step
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14371/117/pipeline-console/?start-byte=0&selected-node=507#log-0
Not in tests, which are not triggered in CI by default.